nuxeo-runtime/nuxeo-connect-standalone/src/main/java/org/nuxeo/launcher/connect/ConnectBroker.java

```diff
@@ -178,7 +179,12 @@ protected LocalPackage getInstalledPackage(String pkgName) {
     }
 
     protected boolean isInstalledPackage(String pkgName) {
-        return service.getPersistence().getActivePackageId(pkgName) != null;
+        try {
+            return service.getPersistence().getActivePackageId(pkgName) != null;
+        } catch (PackageException e) {
+            log.error(e);
```

Comment:
Use `log.error("Failed to check package installation for: " + pkgName, e)` to be informative and have a stack trace (`log.error(e)` uses `e` as the message not the throwable so doesn't show stack trace).